provisioning/roles/utils/redis/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Ensure Redis is installed.
+  apt:
+    name: redis-server
+    state: latest
+  become: true
+
+- name: Ensure Redis' data directory exists.
+  file:
+    path: '{{ redis_data_directory }}'
+    owner: 'redis'
+    group: 'redis'
+    mode: '0755'
+  become: true
+
+- name: Configure Redis' overrides.
+  template:
+    src: 'etc/redis/redis.conf'
+    dest: '/etc/redis/redis.conf'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
```

Comment:
I had many problems with permissions during the development of this.

So my findings were:

- `redis:redis` does not work on these files (I'll retry, though, because it may be due to others factors).
- Right on 0644!
- @ core.conf: it makes it easier to configure defaults in the future and the role will get more parameters to handle Mangaki's needs (if necessary at all). Anyway, I prefer to explicit those settings.